src/DistortableImageOverlay.js

```diff
@@ -211,16 +211,20 @@ L.DistortableImageOverlay = L.ImageOverlay.extend({
   },
 
   _cornerExceedsMapLats: function(zoom, corner, map) {
-    var exceedsTop;
-    var exceedsBottom;
-    if (zoom === 0) {
-      exceedsTop = map.project(corner).y < 2;
-      exceedsBottom = map.project(corner).y >= 255;
-    } else {
-      exceedsTop = map.project(corner).y / zoom < 2;
-      exceedsBottom = map.project(corner).y / Math.pow(2, zoom) >= 255;
+    if (map.options.crs == L.CRS.Simple) {
+      return false;
+    } else {  
+      var exceedsTop;
+      var exceedsBottom;
+      if (zoom === 0) {
+        exceedsTop = map.project(corner).y < 2;
+        exceedsBottom = map.project(corner).y >= 255;
+      } else {
+        exceedsTop = map.project(corner).y / zoom < 2;
+        exceedsBottom = map.project(corner).y / Math.pow(2, zoom) >= 255;
+      }
+      return (exceedsTop || exceedsBottom);      
```

Comment:
```suggestion
      return (exceedsTop || exceedsBottom);
```

Here too. It's quite picky about formatting!